src/components/CreateUserModal.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-buttons slot="start">
+          <ion-button @click="closeModal">
+            <ion-icon :icon="closeOutline" slot="icon-only" />
+          </ion-button>
+        </ion-buttons>
+        <ion-title>{{ $t("Create user") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button fill="clear">{{ $t("Save") }}</ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <ion-item lines="full">
```

Comment:
do we need lines full here. what happens if we remove them?

Reply:
No, we don't need full lines here. I have removed it and it is looking like this :
![localhost_8102_users](https://user-images.githubusercontent.com/82817616/155693734-c4c5f00d-816b-4496-bbf4-47e3ab5f42da.png)